internal/features/stacks/events.go

```diff
@@ -186,6 +186,20 @@ func (f *StacksFeature) decodeStack(ctx context.Context, dir document.DirHandle,
 	}
 	ids = append(ids, parseId)
 
+	metaId, err := f.stateStore.JobStore.EnqueueJob(ctx, job.Job{
+		Dir: dir,
+		Func: func(ctx context.Context) error {
+			return jobs.LoadStackMetadata(ctx, f.store, path)
+		},
+		Type:        operation.OpTypeLoadModuleMetadata.String(),
```

Comment:
I think we should introduce a new job type, so that these jobs won't get merged with module metadata jobs.